src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,768 @@
+from __future__ import annotations
+
+import enum
+import logging
+from argparse import ArgumentParser
+from ast import Tuple
+from dataclasses import dataclass
+from pathlib import Path
+from pprint import pformat, pprint
+from typing import Any, Dict, Iterator, List, Set
+
+import pytorch_lightning as pl
+import torch
+import torchvision.transforms as T
+from PIL import Image
+from torch import Tensor
+
+import requests
+from detectron2.checkpoint import DetectionCheckpointer
+from detectron2.config import get_cfg
+from detectron2.data import MetadataCatalog
+from detectron2.projects.deeplab import add_deeplab_config
+from transformers.models.maskformer import (
+    MaskFormerConfig,
+    MaskFormerForPanopticSegmentation,
+    MaskFormerForPanopticSegmentationOutput,
+    MaskFormerForSemanticSegmentation,
+    MaskFormerForSemanticSegmentationOutput,
+    MaskFormerModel,
+)
+from transformers.models.maskformer.configuration_maskformer import ClassSpec, DatasetMetadata
```

Comment:
```suggestion
from transformers.models.maskformer.configuration_maskformer import ClassSpec, DatasetMetadata
```
To be removed if we add the additional mappings to `PretrainedConfig`.